docs(grid): Update export article (#1607)
telerik/blazor-docs: components/grid/export/csv.md

}
````

## Customization

To customize the exported file, handle the `OnBeforeExport` or `OnAfterExport` events the Grid exposes. 

> The component allows you to control the data set that will be exported. It also provides built-in customization options for the columns such as `Width`, `Title` and more.

Reviewer: ```suggestion
The component allows you to control the data set that will be exported. It also provides built-in customization options for the columns, such as `Width`, `Title`, and more.
```

> For more advanced customization (such as coloring the headers or bolding the titles) the Grid lets you get the `MemoryStream` of the file. Thus, you can customize it using the `SpreadProcessing` or the `SpreadStreamProcessing` libraries that are available with your license.

Reviewer 2: Link [DPL](https://docs.telerik.com/blazor-ui/components/document-processing/overview) or [SpreadProcessing](https://docs.telerik.com/blazor-ui/components/document-processing/spreadsheet) somewhere.

```suggestion
For more advanced customizations (such as coloring the headers, bolding the titles or even changing cell values) the Grid lets you get the `MemoryStream` of the file. Thus, you can customize it using the `SpreadProcessing` or the `SpreadStreamProcessing` libraries that are available with your license.
```

> For more advanced customizations (such as coloring the headers, bolding the titles, or even changing cell values) the Grid lets you get the `MemoryStream` of the file. Thus, you can customize it using the `SpreadProcessing` or the `SpreadStreamProcessing` libraries that are available with your license.

Author: ```suggestion
For more advanced customizations (such as coloring the headers, bolding the titles, or even changing cell values) the Grid lets you get the `MemoryStream` of the file. Thus, you can customize it using the [`SpreadProcessing`](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/overview) or the [`SpreadStreamProcessing`](https://docs.telerik.com/devtools/document-processing/libraries/radspreadstreamprocessing/overview) libraries that are available with your license.
```
